app/frontend/src/Overlay/index.js

```diff
@@ -23,18 +24,25 @@ const Overlay = () => {
   } } = globalSettings || {}
 
   const [ line ] = useCurrentLine()
+  const { lineEnding } = overlay
 
-  const translations = useTranslations( line && [
-    overlay.englishTranslation && LANGUAGES.english,
-    overlay.punjabiTranslation && LANGUAGES.punjabi,
-    overlay.spanishTranslation && LANGUAGES.spanish,
-  ] )
-
-  const transliterations = useTransliterations( line && [
-    overlay.englishTransliteration && LANGUAGES.english,
-    overlay.hindiTransliteration && LANGUAGES.hindi,
-    overlay.urduTransliteration && LANGUAGES.urdu,
-  ] )
+  const translations = customiseLines(
+    useTranslations( line && [
+      overlay.englishTranslation && LANGUAGES.english,
+      overlay.punjabiTranslation && LANGUAGES.punjabi,
+      overlay.spanishTranslation && LANGUAGES.spanish,
+    ] ),
+    { lineEnding },
+  )
```

Comment:
```suggestion
  const translations = mapValues(
    useTranslations( line && [
      overlay.englishTranslation && LANGUAGES.english,
      overlay.punjabiTranslation && LANGUAGES.punjabi,
      overlay.spanishTranslation && LANGUAGES.spanish,
    ] ),
    customiseLine( { lineEnding } ),
  )
```

That doesn't look so bad, I'd say it looks more clear. I think we are going to have many more cases where we'd want to copy a single line and apply these customisations. 

Also, it may be possible that removing vishraams etc can be moved into our new function, as a side note (you don't have to action this).

Reply:
I tried using `mapValues` but can't pass in the value for `lineEnding`

Reply:
ended up modifying lodash function